examples/x86_64.swiftinterface

```diff
@@ -0,0 +1,511 @@
+  public let logger: AzureCore.ClientLogger
+  public let telemetryOptions: AzureCore.TelemetryOptions
+  public let transportOptions: AzureCore.TransportOptions
+  public let dispatchQueue: Dispatch.DispatchQueue?
+  public enum ApiVersion : AzureCore.RequestStringConvertible {
+    case custom(Swift.String)
+    case v20210307
+    public static var latest: AzureCommunicationChat.AzureCommunicationChatClientOptions.ApiVersion {
+      get
+    }
+    public var requestString: Swift.String {
+      get
+    }
+    public init(_ val: Swift.String)
+  }
+  public init(apiVersion: AzureCommunicationChat.AzureCommunicationChatClientOptions.ApiVersion = .latest, logger: AzureCore.ClientLogger = ClientLoggers.default(tag: "AzureCommunicationChat"), telemetryOptions: AzureCore.TelemetryOptions = TelemetryOptions(), transportOptions: AzureCore.TransportOptions? = nil, dispatchQueue: Dispatch.DispatchQueue? = nil)
+}
+public struct ChatThreadItem : Swift.Codable {
+  public let id: Swift.String
+  public let topic: Swift.String
+  public let deletedOn: AzureCore.Iso8601Date?
+  public let lastMessageReceivedOn: AzureCore.Iso8601Date?
+  public init(id: Swift.String, topic: Swift.String, deletedOn: AzureCore.Iso8601Date? = nil, lastMessageReceivedOn: AzureCore.Iso8601Date? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct PhoneNumberIdentifierModel : Swift.Codable {
+  public let value: Swift.String
+  public init(value: Swift.String)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct AddChatParticipantsResult : Swift.Codable {
+  public let invalidParticipants: [AzureCommunicationChat.ChatError]?
+  public init(invalidParticipants: [AzureCommunicationChat.ChatError]? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public struct UpdateChatMessageRequest : Swift.Codable {
+  public let content: Swift.String?
+  public init(content: Swift.String? = nil)
+  public init(from decoder: Swift.Decoder) throws
+  public func encode(to encoder: Swift.Encoder) throws
+}
+public enum IdentifierSerializer {
+  public static func deserialize(identifier: AzureCommunicationChat.CommunicationIdentifierModel) throws -> AzureCommunicationCommon.CommunicationIdentifier
+  public static func assertOneNestedModel(_ identifier: AzureCommunicationChat.CommunicationIdentifierModel) throws
+  public static func serialize(identifier: AzureCommunicationCommon.CommunicationIdentifier) throws -> AzureCommunicationChat.CommunicationIdentifierModel
+}
+extension CommunicationCloudEnvironmentModel {
+  public static var Public: AzureCommunicationChat.CommunicationCloudEnvironmentModel
+  public static var Dod: AzureCommunicationChat.CommunicationCloudEnvironmentModel
+  public static var Gcch: AzureCommunicationChat.CommunicationCloudEnvironmentModel
```

Comment:
What is the purpose of these static variables if the `CommunicationCloudEnvironmentModel` enum already contains these values?

Reply:
No I don't think we need both. Are these static vars being used at all anywhere?

Reply:
they were being referenced inside the IdentifierSerializer but I just updated it to use the generated enum instead